wordpress-mobile/WordPress-iOS
# Blogging Prompts: Update Core Data properties for Bloganuary

Related to #21952

Now that we've switched to the v3 API, this PR cleans up the unused properties `title` and `content` from the `BloggingPrompt` Core Data model. Even in v2 API, these properties weren't used so there shouldn't be any side effects.

Additionally, I've also added a new property, `additionalPostTags` which is a `Transformable` property with a `[String]` type. This will be used for the Bloganuary project in the next PR.

## To test

Smoke test on Blogging Prompt features:

- Make sure the prompt card appears on the Home dashboard.
- Make sure the recent prompts are loaded and showing correctly from the ellipsis menu > View more prompts.
- Tap 'View all answers' and verify that the stream shows the correct daily prompt tag.

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the changes.

3. What automated tests I added (or what prevented me from doing so)
N/A.

PR submission checklist:


UI Changes testing checklist:

N/A.
